server/routes/trial-management/add-panel-members/add-panel-member.controller.js

```diff
@@ -0,0 +1,199 @@
+(function() {
+  'use strict';
+
+  const  _ = require('lodash')
+    , addPanelMembersValidator = require('../../../config/validation/add-panel-members')
+    , { addPanelMembersDAO, availableJurorsDAO } = require('../../../objects/panel')
+    , poolsValidator = require('../../../config/validation/generate-panel-pools')
+    , validate = require('validate.js');
+
+  const countErrors = (tmpErrors) => typeof tmpErrors !== 'undefined' ? Object.keys(tmpErrors).length : 0;
+
+
+  module.exports.getAddPanelMember = function(app) {
+    return function(req, res) {
+      let tmpErrors
+        , tmpFields;
+
+      tmpErrors = _.clone(req.session.errors);
+      tmpFields = _.clone(req.session.formFields);
```

Comment:
these could be done as:

```js
const tmpErrors = ....
const tmpFields = ....
```

instead of splitting into different steps

Reply:
changed